src/qml/QFieldCloudPopup.qml

```diff
@@ -721,4 +729,20 @@ Popup {
     cloudProjectsModel.discardLocalChangesFromCurrentProject(cloudProjectsModel.currentProjectId)
     cloudProjectsModel.downloadProject(cloudProjectsModel.currentProjectId, true)
   }
+
+  function applyAutoPushProjectSettings() {
+    if (cloudProjectsModel.currentProjectId !== '') {
+      const forceAutoPush = iface.readProjectBoolEntry("qfieldsync", "forceAutoPush", false)
+      if (forceAutoPush) {
+        autoPush.forceAutoPush = true
+        autoPush.enabled = false
+        const forceAutoPushIntervalMins = iface.readProjectNumEntry("qfieldsync", "forceAutoPushIntervalMins", 30)
+        cloudProjectsModel.projectSetAutoPushEnabled(cloudProjectsModel.currentProjectId, true)
+        cloudProjectsModel.projectSetAutoPushIntervalMins(cloudProjectsModel.currentProjectId, forceAutoPushIntervalMins)
+      } else {
+        autoPush.forceAutoPush = false
+        autoPush.enabled = true
+      }
+    }
+  }
 }
```

Comment:
Can't we skip all of these and just introduce `cloudProjectsModel.currentProjectData.ForceAutoPushEnabled`? In addition, we can make `cloudProjectsModel.currentProjectData.AutoPushEnabled` always `true` and the rest of the logic is as is in already existing code for `cloudProjectsModel.currentProjectData.AutoPushEnabled`?